codebuild/bin/install_default_dependencies.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+set -ex
+source codebuild/bin/s2n_setup_env.sh
+
+ # Install latest version of clang, clang++, and llvm-symbolizer. Needed for fuzzing.
+if [[ "$TESTS" == "fuzz" || "$TESTS" == "ALL" || "$LATEST_CLAG" == "true" ]] && [[ ! -d "$LATEST_CLANG_INSTALL_DIR" ]]; then
```

Comment:
Typo in the check for LATEST_CLANG environment variable, checks for LATEST_CLAG instead.